packages/react-components/src/components/utils/filtrer-tree/utils.tsx

```diff
@@ -31,7 +31,9 @@ export const listToTree = (list: Entry[], headers: ColumnHeader[]): TreeNode[] =
         if ((entry.parentId !== undefined) && (entry.parentId !== -1)) {
             const parent: TreeNode = lookup[entry.parentId];
             if (parent) {
```

Comment:
Should the condition be included here? In the case of the circular reference should the node be considered a root or not? Otherwise does it mean the node just disappears from the tree?